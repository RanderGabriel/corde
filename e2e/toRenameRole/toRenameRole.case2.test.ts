import { runCLI } from "../cliRunner";
import Utils from "../testUtils";

it("tests should be not be sucessfull", async () => {
  const command = Utils.buildCommandWithConfigPath("toRenameRole", "bot_case2.test.ts");
  const results = await cli.exec(command);
  expect(results.statusCode).toEqual(1);
  expect(results.stdout).toMatchSnapshot();
});
