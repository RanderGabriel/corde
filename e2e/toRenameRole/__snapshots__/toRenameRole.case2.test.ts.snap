// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests should be not be sucessfull 1`] = `
"
 FAIL   e2e\\\\toRenameRole\\\\__cordeTest__\\\\bot_case2.test.ts   100ms
   x <empty test name>   100ms
    
    expected: role 'old-role-name' to be renamed to batata
    received: name was not changed

    at toRenameRole e2e\\\\toRenameRole\\\\__cordeTest__\\\\bot_case2.test.ts:8:49
    at lib\\\\src\\\\clausures\\\\test.js:31:13
    at Queue.executeAsync (lib\\\\src\\\\data-structures\\\\queue.js:99:27)

Test Files: 1 failed. 1 total
Tests:      1 failed. 1 total
Time:       100ms
"
`;
